model_utils/fields.py

```diff
@@ -126,6 +126,33 @@ def deconstruct(self):
         return name, path, args, kwargs
 
 
+class PersistentMonitorField(MonitorField):
+    """
+    Like MonitorField, but once the timestamp for the first change is recorded
+    it is never set again
+    """
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault('default', now())
```

Comment:
This isn't right, is it? Sets the default value to the time whenever the model is imported, and won't change until the server restarts.